docs/SRS-Volere/SRS.tex

```diff
@@ -171,15 +171,27 @@ \subsection{Longevity Requirements}
 
 \section{Operational and Environmental Requirements}
 \subsection{Expected Physical Environment}
-\lips
+\begin{enumerate} [{OE-PE}1.]
+  \item Application should be functional in City of Hamilton, Water Division sites and offices.
+\end{enumerate}
 \subsection{Wider Environment Requirements}
-\lips
+\begin{enumerate} [{OE-WE}1.]
+  \item Application should be functional on Mobile and Desktop web browser environments.
+  \item Application should be able to run on any device with a modern web browser.
+\end{enumerate}
 \subsection{Requirements for Interfacing with Adjacent Systems}
-\lips
+\begin{enumerate} [{OE-IAS}1.]
+  \item Application should integrate with existing SharePoint repositories.
+\end{enumerate}
 \subsection{Productization Requirements}
-\lips
+N/A
 \subsection{Release Requirements}
-\lips
+\begin{enumerate} [{OE-REL}1.]
+  \item A change-log should be generated with every release documenting changes in
```

Comment:
This section should define what is considered to be a release. Is it every commit, every deliverable, every revision, every deployment? Also, I think changelog is just one word, no need for the hyphen

Reply:
Added definition, fixed spelling.